config/localization/de/app.json

```diff
@@ -186,7 +186,6 @@
       "PERPETUALS": "Perpetuals",
       "PNL": "G&V",
       "POSITION": "Position",
-      "POWERED_BY": "Powered By",
```

Comment:
deleted non-en translations of `POWERED_BY` so that it could be retranslated in caps lock - not sure if this is the right protocol - should I keep them in instead?